pyproject.toml

```diff
@@ -38,6 +38,10 @@ dependencies = [
 ]
 dynamic = ["version"]
 
+[tool.setuptools.packages.find]
+where = ["."]  # list of folders that contain the packages (["."] by default)
+include = ["gbasis"]  # package names should match these glob patterns (["*"] by default)
```

Comment:
Specifically this line should pass the initial setup of the package that will run action for building and updating website later